src/jackdaw/serdes/avro.clj

```diff
@@ -505,14 +505,17 @@
   {"schema.registry.url" registry-url})
 
 (defn- serializer [schema->coercion serde-config]
-  (let [{:keys [registry-client registry-url avro-schema read-only? key?]} serde-config
-        base-serializer (KafkaAvroSerializer. registry-client)
+  (let [{:keys [registry-client registry-url avro-schema read-only? key?
+                serializer-properties]} serde-config
+        base-config (-> (base-config registry-url)
```

Comment:
Can we call the left hand side var `serializer-config` or `config` instead of `base-config`?